pages/core-api/_meta.json

```diff
@@ -8,13 +8,13 @@
     "title": "Safe{Core} Infrastructure"
   },
   "api-overview": "Overview",
-  "how-to-use-api-keys": "How to use API keys",
```

Comment:
The reason to be there is because API keys are not limited to the Transaction Service api and can be used in any service son we avoided the specific Transaction Service section